### PR TITLE
Nik john/add material ui ssr

### PR DESCRIPTION
**Pull request by nik-john**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

Material UI stylesheet also need to be rendered on the server side. This PR adds this change and adds a sample component as per [Official Material UI Docs](https://github.com/mui-org/material-ui/tree/master/examples/nextjs)

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,16 +1,9 @@
-import * as React from 'react';
+import React from 'react';
 import SomeComponent from 'client/components/SomeComponent';
-import styled from 'styled-components';
-
-const Title = styled.h1`
-  font-size: 50px;
-  color: ${({ theme }) => theme.colors.primary};
-`;
```
Comment (timmyichen) [inline pages/index.tsx]: can we add back in some example of using styled components for future contributors?

**Comment by ThomasRoest** [comment]
thoughts about folder structure? My preference would be

```
MyComponent
      --- index.tsx
      --- styles.ts
```

File: pages/index.tsx (hunk 1 above)
Comment (nik-john) [reply pages/index.tsx]: @timmyichen Yup, done.

**Comment by nik-john** [comment]
@ThomasRoest Agreed. Not sure why we didn't do that already, but I've made the change

**Pull request merged**